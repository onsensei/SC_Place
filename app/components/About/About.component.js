import React, {Component} from 'react';
import styles from './About.style';
import {Text, View} from 'react-native';

export default class About extends Component {
  render () {
    return (
      <View style={styles.container}>
        <Text style={styles.welcome}>
          Welcome to About page
        </Text>
      </View>
    );
  }
}
