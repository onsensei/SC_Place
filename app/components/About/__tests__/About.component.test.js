import About from '../About.component';

// Note: test renderer must be required after react-native.
import React from 'react';
import renderer from 'react-test-renderer';

describe('About', () => {
  let component;

  beforeEach(() => {
    component = <About/>;
  });

  it('renders correctly', () => {
    const tree = renderer.create(component);
    expect(tree).toMatchSnapshot();
  });
});
