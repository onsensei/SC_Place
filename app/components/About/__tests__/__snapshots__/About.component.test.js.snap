// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    Welcome to About page
  </Text>
</View>
`;
